### ISSUE TITLE
Unhighlighted Navbar Menu text doesn't meet AAA from WCAG

### ISSUE THREAD

**Issue by codyduong** [codyduong/codyduong.dev#9]
The unhighlighted text color doesn't have enough contrast to be compliant
![2022-12-06-140638_324x536_scrot](https://user-images.githubusercontent.com/24193268/206011829-cdcc2a20-e844-4374-a140-a1b84ff67c2b.png)
